src/lean_spec/subspecs/containers/block/block.py

```diff
@@ -56,14 +57,29 @@ class Block(Container):
     """The block's payload."""
 
 
-class SignedBlock(Container):
-    """A container for a block and the proposer's signature."""
+class BlockAndVote(Container):
```

Comment:
As what we have in a block is attestation, I'd advocate for `BlockWithAttestation`.